pkg/build/buildscheme/scheme.go

```diff
@@ -33,12 +32,12 @@ func init() {
 	legacy.InstallInternalLegacyBuild(annotationDecodingScheme)
 	utilruntime.Must(buildv1helpers.Install(annotationDecodingScheme))
 	annotationDecoderCodecFactory := serializer.NewCodecFactory(annotationDecodingScheme)
-	Decoder = annotationDecoderCodecFactory.UniversalDecoder(buildapi.SchemeGroupVersion)
+	Decoder = annotationDecoderCodecFactory.UniversalDecoder(buildv1.GroupVersion)
```

Comment:
@deads2k i guess this is expected... we don't have to decode to internal anymore